src/main/webapp/app/shared/util/firebase/firebase-reviewable-crud-store.ts

```diff
@@ -0,0 +1,55 @@
+import { IRootStore } from 'app/stores';
```

Comment:
didn't review. I assume this is cover over from the crud store

Reply:
Yeah, this was copied over and placed in its own file.